dist/index.js

```diff
@@ -13181,30 +13180,7 @@ function merged(configuration) {
                 return;
             }
         }
-        const newOutput = points.setBalance(balanceSheet, user, balance);
-        try {
-            toml.parse(newOutput);
-        }
-        catch (_g) {
-            return Promise.reject(`setBalance resulted in invalid output: ${newOutput}`);
-        }
-        const octokit = github.getOctokit(core.getInput("token"));
-        const fileContentsParams = {
-            owner: (_b = (_a = github.context.payload.repository) === null || _a === void 0 ? void 0 : _a.owner) === null || _b === void 0 ? void 0 : _b.login,
-            repo: (_c = github.context.payload.repository) === null || _c === void 0 ? void 0 : _c.name,
-            path: ".github/gbp-balances.toml",
-        };
-        const sha = yield octokit.repos
-            .getContent(fileContentsParams)
-            .then((contents) => {
-            const data = contents.data;
-            return Array.isArray(data) ? undefined : data.sha;
-        })
-            .catch(() => {
-            // Most likely 404
-            return undefined;
-        });
-        yield octokit.repos.createOrUpdateFileContents(Object.assign(Object.assign({}, fileContentsParams), { message: `Updating GBP from PR #${pullRequest.number} [ci skip]`, content: Buffer.from(newOutput, "binary").toString("base64"), sha }));
+        yield points.writeBalanceOf(octokit, configuration.branch, (_e = (_d = github.context.payload.repository) === null || _d === void 0 ? void 0 : _d.owner) === null || _e === void 0 ? void 0 : _e.login, (_f = github.context.payload.repository) === null || _f === void 0 ? void 0 : _f.name, `Updating GBP from PR #${pullRequest.number} [ci skip]`, user.id, balance);
```

Comment:
what he fck

Reply:
its the compiled dist you dip, you need it for github actions because it doesnt support typescript

Reply:
this entire file is `ncc build src/index.ts -o dist`

Reply:
😂